src/core/utils/data-transform.ts

```diff
@@ -108,23 +107,25 @@ export async function joinData(
         for (const d of data) {
             const chrName = d[from.chromosomeField];
             const chromPosition = d[from.genomicField];
-            d[from.genomicField] = chromSizes.interval[chrName]?.[0] + +chromPosition;
+            d[from.genomicField] = (chromSizes.interval[chrName]?.[0] ?? 0) + +chromPosition;
```

Comment:
This was the main reason why the `coord` was `NaN` before. In the minimal example, genome `assembly` is not defined. In this case, Gosling uses the default assembly, which is a human genome (hg38). However, since the chromosome names defined in the 3D genome model (A, B, C, ...) do not match the chromosome names for the human genome (chr1, chr2, chr3, ...), `chromSizes.interval[chrName]?.[0]` is `NaN`.

By adding `?? 0`, the parsed genomic positions (`coord`) are not `NaN` anymore.